src/Sitemap/SitemapUrl.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace OHMedia\PageBundle\Sitemap;
+
+class SitemapUrl
+{
+    public readonly float $priority;
+
+    public function __construct(
+        public readonly string $path,
+        public readonly \DateTimeInterface $lastmod
+    ) {
+        $priority = 1.0;
+
+        if ($this->path) {
+            $parts = explode('/', $this->path);
+
+            $priority -= 0.1 * count($parts);
+        }
+
+        $this->priority = $priority;
```

Comment:
Priority isn't supposed to be used by search engines, so I'm just putting a linearly decreasing value in there based on how many URL parts there are.